lib/elixir/lib/node.ex

```diff
@@ -72,7 +72,8 @@ defmodule Node do
   The result returned when the argument is a list, is the list of nodes
   satisfying the disjunction(s) of the list elements.
 
-  See http://www.erlang.org/doc/man/erlang.html#nodes-1 for more info.
+  For more information, see
+  [`:erlang.nodes/1`](http://www.erlang.org/doc/man/erlang.html#nodes-1).
```

Comment:
We should just keep it succinct and correct. We can figure out how to make it work on the terminal on the side. So we can remove the links. :)